app/helpers/green_lanes_helper.rb

```diff
@@ -22,6 +22,26 @@ def all_exemption_questions_replied?(questions_count)
     questions_count == questions_replied_count
   end
 
+  def render_exemptions_or_no_card(category, exemptions_met, no_exemptions)
+    if no_exemptions
```

Comment:
You could just condense this chipolata
```suggestion
    if no_exemptions || exemptions_met.empty?
```